include/sfx.h

```diff
@@ -459,4 +452,16 @@ enum Sfx {
     SFX_EXPLODE_SMALL, // Zombie death explosion
 };
 
+// MAD uses an earlier build and has different sfx IDs
+enum MAD_Sfx {
```

Comment:
The way this should be done is
```
#ifdef VERSION_BETA
enum Sfx {
  SFX_WEAPON_STAB_B = 0x630,
...
#else
enum Sfx {
...
#endif
```

This will get if of `#ifdef VERSION_BETA` in several places. The plan is to deduplicate things more in the future so the same functions ought to have the same code